### PR TITLE
release v1.0.0

### PR DESCRIPTION
**Pull request by nejisama**
### Issues associated with this PR

Your PR should present related issues you want to solve.

### Solutions
You should show your solutions about the issues in your PR, including the overall solutions, details and the changes. At this time, Chinese is allowed to describe these.

### UT result
Unit Test is needed if the code is changed, your unit test should cover boundary cases, corner cases, and some exceptional cases. And you need to show the UT result.

### Benchmark
If your code involves the processing of every request, you should give the Benchmark Result.

### Code Style
+ Make sure `Goimports` has run
+ Show `Golint` result

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,21 @@
...
 
+## v1.0.0
+
+### Changes
+
+- Protocol: bolt support GoAway (#1993) [@z2z23n0](https://github.com/z2z23n0)
+- Protocol: HTTP health check support more configurations (#1999) [@dengqian](https://github.com/dengqian)
+- Add new Admin API for query MOSN version (#2002) [@songzhibin97](https://github.com/songzhibin97)
+- Exit code change to 2 when mosn start failed in upgrade mode (#2006) [@doujiang24](https://github.com/doujiang24)
+- Add a new function to check wether MOSN is in active upgrading state (#2003) [@doujiang24](https://github.com/doujiang24)
```
Comment (doujiang24) [inline CHANGELOG.md:11]: typo: wether => whether

Comment (nejisama) [reply CHANGELOG.md]: fixed

**Review by antJack — APPROVED** [review]
LGTM

File: README_ZH.md [hunk 2]
```diff
@@ -13,28 +13,61 @@ MOSN（Modular Open Smart Network）是一款主要使用 Go 语言开发的云
...
+    + 支持基于 prometheus 的 metrics 格式数据
+    + 支持可配置的 AccessLog
+    + 支持可扩展的 Admin API
+    + 集成 [Holmes](https://github.com/mosn/holmes)，自动监控 pprof
++ TLS
+    + 支持多证书匹配模式、支持 TLS Inspector 模式
+    + 支持基于 SDS 的动态证书获取、更新机制
+    + 支持可扩展的证书获取、更新、校验机制
+    + 支持基于 CGo 的国密套件
++ 进程管理
```
Comment (wangfakang) [inline README_ZH.md:55]: align

**Review by doujiang24 — APPROVED** [review]
